ui/src/shared/reducers/app.ts

```diff
@@ -28,6 +29,7 @@ const initialState: AppState = {
     showTemplateControlBar: false,
     timeZone: 'Local',
     navBarState: 'collapsed',
+    notebookMiniMapState: 'expanded',
```

Comment:
dope

Reply:
I learned from @121watts to use more descriptive values instead of booleans